v-next.sql

```diff
@@ -1258,3 +1258,20 @@ insert into openchpl.certification_result_test_tool (certification_result_id, te
     ((select cr.certification_result_id from openchpl.certification_result cr where cr.certified_product_id = 8299 and cr.certification_criterion_id = (select cc.certification_criterion_id from openchpl.certification_criterion cc where cc.number = '170.314 (c)(1)')), (select tt.test_tool_id from openchpl.test_tool tt where tt.name = 'Cypress' and tt.deleted = false), -1, '2.4.1'),
     ((select cr.certification_result_id from openchpl.certification_result cr where cr.certified_product_id = 8300 and cr.certification_criterion_id = (select cc.certification_criterion_id from openchpl.certification_criterion cc where cc.number = '170.314 (c)(1)')), (select tt.test_tool_id from openchpl.test_tool tt where tt.name = 'Cypress' and tt.deleted = false), -1, '2.4.1'),
     ((select cr.certification_result_id from openchpl.certification_result cr where cr.certified_product_id = 8370 and cr.certification_criterion_id = (select cc.certification_criterion_id from openchpl.certification_criterion cc where cc.number = '170.314 (c)(1)')), (select tt.test_tool_id from openchpl.test_tool tt where tt.name = 'Cypress' and tt.deleted = false), -1, '2.4.1');
+
+-- OCD-2377 -  Cleanup of unused API keys
+ALTER TABLE openchpl.api_key ADD COLUMN last_used_date timestamp without time zone DEFAULT now();
```

Comment:
Does this work multiple times? Does the function creation work multiple times?

Should also be in v-dev.sql

Reply:
This stuff should only be in v-dev and not v-next at all right? Due to STG being frozen?